UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/AzureWebJobHealthCheck.cs

```diff
@@ -0,0 +1,36 @@
+﻿
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using UKHO.ExternalNotificationService.Common.Logging;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    public class AzureWebJobHealthCheck : IHealthCheck
+    {
+        private readonly IAzureWebJobHealthCheckService _azureWebJobHealthCheckService;
+        private readonly ILogger<AzureWebJobHealthCheck> _logger;
+
+        public AzureWebJobHealthCheck(IAzureWebJobHealthCheckService azureWebJobHealthCheckService,
+                                       ILogger<AzureWebJobHealthCheck> logger)
+        {
+            _azureWebJobHealthCheckService = azureWebJobHealthCheckService;
+            _logger = logger;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            HealthCheckResult healthCheckResult = await _azureWebJobHealthCheckService.CheckHealthAsync();
```

Comment:
Forward the CancellationToken parameter to methods CheckHealthAsync()  as:
HealthCheckResult healthCheckResult = await _azureWebJobHealthCheckService.CheckHealthAsync(cancellationToken);

Reply:
Changes done